Fix bouncy scrolling on mobile web

darrenvong opened the pull request:
Considering I can just about reproduce this on mobile Chrome (Firefox works fine for me on Android even without this fix...), fixing the map overflow on the edit profile page seems to have done the trick.

Might be good to have an iPhone user to check and see how it behaves on Safari too? 🤷 

Fixes #817

darrenvong committed:
fix: change width to maxWidth to stop map overflowing

lucaslcode commented on app/frontend/src/components/EditUserLocationMap.tsx:
```diff
@@ -24,7 +24,7 @@ const useStyles = makeStyles({
     height: 200,
     position: "relative",
-    width: 400,
+    maxWidth: 400,
```
really going all out on this PR

lucaslcode commented:
Let's see if it is fixed for jesse

darrenvong committed:
fix: tweak styling to fix map on page

darrenvong committed:
fix: change message list width back to 100%

This seems unnecessary with just three tabs at the moment.

The overflow (horizontal scrolling) it causes on small screens is making
scrolling "bouncy"...
